network/endpoint.go

```diff
@@ -21,6 +21,8 @@ type endpoint struct {
 	MacAddress  net.HardwareAddr
 	IPAddresses []net.IPNet
 	Gateways    []net.IP
+	DNS         DNSInfo
```

Comment:
can we move Data to last field in structure..also plz group same datatypes together

Reply:
Done